site/content/entry/yuzu-progress-report-apr-2021/index.md

```diff
@@ -0,0 +1,227 @@
+By [avoiding spin loops](https://github.com/yuzu-emu/yuzu/pull/6162) in the video core, more idle time is achieved with no performance loss.
+This means the GPU can enter `sleep` state more often, resulting in lower temperatures/power consumption, better battery life, or allowing the scheduler to take on other tasks waiting in the background, improving system responsiveness.
+No more burned thighs while on the pause menu, laptop users!
+
+[Joshua-Ashton](https://github.com/Joshua-Ashton), the Frog Master from [DXVK](https://github.com/doitsujin/dxvk), brought us a couple of Vulkan fixes and improvements.
+
+First on the list is avoiding a device loss (GPU locks up and the driver restarts it) by [checking the return value of vkAcquireNextImageKHR.](https://github.com/yuzu-emu/yuzu/pull/6180)
+
+Joshua also [enabled the individual features](https://github.com/yuzu-emu/yuzu/pull/6181) from `VK_EXT_robustness2`, instead of just enabling the extension.
+
+And finally, [fixing a wrong offset for null vertex buffers](https://github.com/yuzu-emu/yuzu/pull/6182) to properly follow the Vulkan specification. 
+You never know when a driver will behave in erratic ways. Fixes like this allow developers to report bugs to the vendor’s driver development teams, thanks to confirming everything is within specs. Any resulting error falls under responsibility of the driver teams.
+
+Not only OpenGL had BGR issues, it turns out Vulkan can also suffer from blue madness when a game uses the `A1B5G5R5_UNORM` texture format.
+
+[epicboy](https://github.com/ameerj) [swapped the Red and Blue channels](https://github.com/yuzu-emu/yuzu/pull/6238) of this format, fixing colours in games like `Pokémon Let’s Go Eevee/Pikachu`, `Shantae: Risky’s Revenge`, and many others!
+You can see the result in the following pictures, before (left) and after (right).
+
+{{< single-title-imgs
+    "I didn’t want Pokémon Blue! (Pokémon Let's Go, Eevee!)"
+    "./pokebug.png"
+    "./pokefix.png"
+  >}}
+
+{{< single-title-imgs
+    "That’s definitely not Vivi (Shantae: Risky’s Revenge)"
+    "./shanbug.png"
+    "./shanfix.png"
+  >}}
+
+## Linux Build System
+
+[toastUnlimited](https://github.com/lat9nq) has been periodically improving yuzu’s build system for our Linux user base. 
+Not all our users want or are able to use the official AppImage, so not only did our build instructions needed to improve, but it was also equally important to refine the environment necessary to get yuzu running — namely, the dependencies, build process, etc.
+
+The first step was to [make SDL2](https://github.com/yuzu-emu/yuzu/pull/6204) [an external dependency.](https://github.com/yuzu-emu/yuzu/pull/6207)
+This would allow us to ship the latest version without having to face Linux distributions incompatibilities due to different build flags (different characteristics being enabled), or outright outdated versions that aren’t able to support all of the required features that our input asks for.
```

Comment:
```suggestion
This would allow us to ship the latest version without having to face Linux distributions' incompatibilities due to different build flags (different characteristics being enabled), or outright outdated versions that aren’t able to support all of the required features that our input asks for.
```